src/setup/tsconfig.ts

```diff
@@ -13,7 +13,7 @@ export function getTypeScriptConfig(buildStepConfig: any): any {
       listFiles: true,
       declaration: true,
       removeComments: true,
-      lib: ['es2015', 'dom'],
+      lib: ['es2017', 'dom'],
```

Comment:
Ich denke hier können wir als lib ruhig `esnext` angeben. Dadurch wird nicht die Ausgabe beeinflusst - nur festgelegt, dass wir in der neuesten TypeScript-Version schreiben falls keine Einschränkung durch die `tsconfig` getroffen wurde.